feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/moduleselection/ModuleSelectionFragment.kt

```diff
@@ -213,11 +223,11 @@ internal class ModuleSelectionFragment : Fragment(R.layout.fragment_sync_module_
         )
 
         editText?.let {
-                it.typeface = try {
-                    ResourcesCompat.getFont(requireContext(), IDR.font.muli)
-                } catch (ex: Exception) {
-                    Typeface.DEFAULT
-                }
+            it.typeface = try {
```

Comment:
Just curious - is this still required with the styling fixes?

Reply:
@luhmirin-s Can you merge this PR now? 
And when @alexandr-simprints is back, he can review your question later.